apps/preview/next/pages/showcases/Card/CategoryCard.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfLink } from '@storefront-ui/react';
+import men from '@assets/men_category.png';
+import women from '@assets/women_category.png';
+import kid from '@assets/kid_category.png';
+
+const categories = [
+  {
+    title: `Women`,
+    image: women.src,
+  },
+  {
+    title: `Men`,
+    image: men.src,
+  },
+  {
+    title: `Kid`,
+    image: kid.src,
+  },
+];
+export default function CategoryCard() {
+  return (
+    <div className="flex flex-wrap gap-4 lg:gap-6 lg:flex-no-wrap">
+      {categories.map(({ title, image }) => (
+        <div
+          className="relative min-w-[180px] first-letter:flex flex-col max-w-[240px] justify-center group"
```

Comment:
```suggestion
        <div
          className="relative min-w-[180px] first-letter:flex flex-col max-w-[240px] group"
```
With current design when width of container is width of image `justify-center` is not needed because on title there is as well `justify-center`

Reply:
question: `first-letter:flex` is not used here, I mean I see no changes when its there or not, maybe I'm missing something :(

Reply:
right removed